### DISCUSSION TITLE
Using spacytextblob in portuguese

### DISCUSSION THREAD

**Discussion by finhopv** [explosion/spaCy#10631] [Help: Coding & Implementations] [answered]
Hello people i trying spacytextblob for sentiment analisys in portuguese, but no working!
when i use 'en' work but when i change for pt, return 0 in polarity and subjectivity. nonetheless i had download 'pt_core_news_sm'

**Comment by polm** [comment, marked as answer]
spacytextblob is a third-party package, so in general you should ask this question at [their repo](https://github.com/SamEdwardes/spacytextblob). 

In this case the issue is simply that textblob only supports English, see https://github.com/SamEdwardes/spacytextblob/issues/14.
